schema/table.go

```diff
@@ -167,6 +179,35 @@ func (tt Tables) ValidateColumnNames() error {
 	return nil
 }
 
+// this will filter the tree in-place
+func (t *Table) filterDfs(tables, skipTables []string) *Table {
+	var matched bool
+	for _, includeTable := range tables {
+		if glob.Glob(includeTable, t.Name) {
+			matched = true
+			break
+		}
+	}
+	for _, skipTable := range skipTables {
+		if glob.Glob(skipTable, t.Name) {
+			return nil
```

Comment:
With this I think we can do the skip `for` loop first and the `tables` for later